react-native-cli/java-keywords.js

```diff
@@ -0,0 +1,33 @@
+'use strict';
+
+/**
+ * Copyright (c) 2015-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ */
+
+const KEYWORDS = [
+  'ABSTRACT',       'ASSERT',       'BOOLEAN',
+  'BREAK',          'BYTE',         'CASE',
+  'CASE',           'CATCH',        'CHAR',
+  'CLASS',          'CONST',        'CONTINUE',
+  'DEFAULT',        'DO',           'DOUBLE',
+  'ELSE',           'ENUM',         'EXTENDS',
+  'FINAL',          'FINALLY',      'FLOAT',
+  'FOR',            'GOTO',         'IF',
+  'IMPLEMENTS',     'IMPORT',       'INSTANCEOF',
+  'INT',            'INTERFACE',    'LONG',
+  'NATIVE',         'NEW',          'PACKAGE',
+  'PRIVATE',        'PROTECTED',     'PUBLIC',
+  'RETURN',         'SHORT',        'STATIC',
+  'STRICTFP',       'SUPER',         'SWITCH',
+  'SYNCHRONIZED',   'THIS',         'THROW',
+  'THROWS',         'TRANSIENT',    'TRY',
+  'VOID',           'VOLATILE',     'WHILE',
+  'REACT'
```

Comment:
I think a single column here would be better as it would be easier to edit/add others to the list if needed. Also, perhaps it can just be named "reserved-words.js", or something similar, especially since "React" is in there.